device/vxlan/destinationipmode.yaml

```diff
@@ -1,6 +1,6 @@
 components:
   schemas:
-    DestinationIPMode:
+    Vxlan.V4Tunnel.DestinationIPMode:
       description: >-
         Communication Mode between the VMs, choice as either multicast or unicast
```

Comment:
Perhaps use this description:
Communication mode between the VMs (representing the VXLAN endpoints), either unicast or multicast.

Reply:
updated